src/Models/Forms/Field.php

```diff
@@ -24,6 +24,6 @@ public function __construct(array $data)
     $this->title = $data['title'];
     $this->type = $data['type'];
     $this->properties = $data['properties'];
-    $this->validations = $data['validations'];
+    $this->validations = isset($data['validations']) ? $data['validations'] : [];
```

Comment:
This might be cleaner with null coalescing i.e. `$this->validations = $data['validations'] ?? [];`